docs/Getting-Started.md

```diff
@@ -174,3 +174,18 @@ Here is another example with a few function signatures:
 		new(func(string) int),
 	)
 ```
+
+Functions taking `context.Context` as first argument are also accepted:
+
+```go
+	username := expr.FunctionWithContext(
+		"username",
+		func(params ...any) (any, error) {
```

Comment:
Same here, the expr.FunctionWithContext can be with signature of

    func (ctx, params...) any, error